src/status_im2/subs/contact.cljs

```diff
@@ -93,21 +93,20 @@
        vals)))
 
 (re-frame/reg-sub
- :contacts/sorted-and-grouped-by-first-letter
```

Comment:
This is still used in some parts of the codebase in develop, and it's currently throwing an error, why has it been removed? is there a replacement to be used or is just a mistake?

Reply:
@flexsurfer I think you shared with me the implementation of this method? Did you rename it by mistake?